src/classes/devices/acaia/acaia.ts

```diff
@@ -355,10 +352,10 @@ export class AcaiaScale {
     return false;
   }
 
-  private async handleNotification(value: ArrayBuffer) {
+  private handleNotification(value: ArrayBuffer) {
     if (this.connected) {
       this.worker.addBuffer(value);
-      await this.heartbeat();
```

Comment:
this one is important, we shouldn't await here to reply with heartbeat on every notification, if we don't do it in a timely manner the scales stop sending weight data.